### DISCUSSION TITLE
macOS Sequoia support is ready, but...

### DISCUSSION THREAD

**Discussion by VisualEhrmanntraut** [ChefKissInc/NootRX#60] [Announcements]
I'm still waiting for Acidanthera to add the Sequoia constants to Lilu. So I can't actually add them in properly. For the meantime, have some screenshots.

![telegram-cloud-photo-size-5-6287266603456380288-y](https://github.com/ChefKissInc/NootRX/assets/30368284/c6fd419e-a7e6-4ef3-9714-4387fce4629f)

![2024-06-13 03 07 31](https://github.com/ChefKissInc/NootRX/assets/30368284/2e1f928c-94fe-4bf6-b354-8c455d81fb74)

**Comment by wander555** [comment]
come on!!!

**Reply by VisualEhrmanntraut** [reply]
Yeah, I'm still waiting on them to add it. :-|

**Comment by VisualEhrmanntraut** [comment]
macOS Sequoia support has been added to NootedRed and NootRX as the constants have been added to Lilu a few moments ago.
